docs/events/timeline-ui-overview.asciidoc

```diff
@@ -2,84 +2,72 @@
 = Investigate events in Timeline
 
 Use Timeline as your workspace for investigations and threat hunting.
-Data from multiple indices can be added to a Timeline, which enables
-investigating complex threats.
+You can add data from multiple indices to a Timeline to facilitate advanced investigations.
 
-You can drag or send items of interest to Timeline to create the query you need
-to investigate an alert or event. You can add items from tables and histograms
+You can drag or send items of interest to a Timeline to create the desired query. You can add items from tables and histograms
```

Comment:
I believe the second sentence (about adding items from tables and histograms) is an example of where users can get data to add to a Timeline. Also, what do you think about switching "items" with "fields" in that second sentence? Either can be used, but "fields" seems more descriptive of the object that users would be dropping into the Timeline query.